ack/setup

```diff
@@ -1,3 +1,3 @@
-#!/bin/bash
+#! /usr/bin/env bash
```

Comment:
Why? Did you check, e.g., https://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my ?

Reply:
To have it to be consistent. As in your link, one is not better than the other. It is or a static path (`/bin/bash`) or one which is resolved based on your path (`/usr/bin/env bash`)

Reply:
IMHO `/usr/bin/env` based formulations are more generalized. The effect can be seen predominantly in a virtualenv setting for python2 and python3. Having just `/usr/bin/env python` automatically resolves the executable based on the environment. 

However I have no knowledge of whether something like virtualenv can be set for something like bash but I like this formulation better.

Reply:
@ar13pit if you approve? Could review the PR?